src/addon/cc-addon-option.js

```diff
@@ -7,7 +7,7 @@ import { i18n } from '../lib/i18n.js';
 /**
  * A component that allows to enable or disable an add-on option.
  *
- * * 🎨 default CSS display: `block`
+ * * 🎨 default CSS display: `flex`
```

Comment:
You used grid in the code.
BTW, I have a branch to automatically get this from code but it's fragile so for now, we need to be thorough.

Reply:
I was sure it was flex..